.github/scripts/split_spec.sh

```diff
@@ -6,9 +6,15 @@ foascli versions -s openapi-foas.json -o ./openapi/v2/versions.json --env "${tar
 
 echo "Running FOAS CLI split command with the following --env=${target_env:?} and -o=./openapi/v2/openapi.json"
 
+# Generate one OAS per version based on the versioning extensions
 foascli split -s openapi-foas.json --env "${target_env:?}" -o ./openapi/v2/openapi.json --format all
-mv -f "openapi-foas.json" "./openapi/v2.json"
-mv -f "openapi-foas.yaml" "./openapi/v2.yaml"
+
+# Filters out the current foas, removing all extensions that are not related with versioning
+foascli filter -s openapi-foas.json --env "${target_env:?}" -o ./openapi/v2/v2.json --format all
```

Comment:
why do we move the v2.json under the v2 folder?

Reply:
good catch 👁️ we shouldn't!